README.md

```diff
@@ -943,20 +966,20 @@ Content-Type: application/json
-| redirect_url        | false     | A valid URL                                                                       |
-| webhook_url         | true      | A valid POST API URL, Detailed response will be sent here for success or failures |
-| purpose             | true      | The reason for the transaction                                                    |
-| phone_override      | false     | If you wanted to update your phone number                                         |
-| document_required   | true      | The document for which you wanted to get the details                              |
-| { ITR               | false     | Specify for how many years of ITR you wanted to fetch                             |
-| 26AS }              | false     | Specify for how many years of 26AS you wanted to fetch                            |
-| phone               | true      | Phone number you wanted to use at our platform                                    |
-| pan                 | true      | PAN number linked to ITR portal                                                   |
-| dob                 | true      | Date of birth of the PAN holder in DD-MM-YYYY format                              |
-| pdf_required        | false     | Whether you need PDF of ITR fetched                                               |
+| Parameters        | Mandatory | Description/Value                                                                 |
+| ----------------- | --------- | --------------------------------------------------------------------------------- |
+| mode              | true      | REDIRECT or POPUP                                                                 |
+| redirect_url      | false     | A valid URL                                                                       |
+| webhook_url       | true      | A valid POST API URL, Detailed response will be sent here for success or failures |
+| purpose           | true      | The reason for the transaction                                                    |
+| phone_override    | false     | If you wanted to update your phone number                                         |
+| document_required | true      | The document for which you wanted to get the details                              |
+| { ITR             | false     | Specify for how many years of ITR you wanted to fetch                             |
+| 26AS }            | false     | Specify for how many years of 26AS you wanted to fetch                            |
+| phone             | true      | Phone number you wanted to use at our platform                                    |
+| pan               | true      | PAN number linked to ITR portal                                                   |
+| dob               | true      | Date of birth of the PAN holder in DD-MM-YYYY format                              |
+| pdf_required      | false     | Whether you need PDF of ITR fetched                                               |
```

Comment:
Please ignore this part, it just formatting change. No need to review this part.